lib/msal-browser-poc/rollup.config.js

```diff
@@ -0,0 +1,76 @@
+import typescript from "rollup-plugin-typescript2";
+import { uglify } from "rollup-plugin-uglify";
+import resolve from "rollup-plugin-node-resolve";
+import pkg from "./package.json";
+
+const libraryHeader = `/*! ${pkg.name} v${pkg.version} ${new Date().toISOString().split('T')[0]} */`;
+const useStrictHeader = `'use strict';`;
+const fileHeader = `${libraryHeader}\n${useStrictHeader}`;
+
+export default [
```

Comment:
looking forward to hearing / reading your thoughts on why roll up may be a good choice for the lib